### DISCUSSION TITLE
Online meetings that don't provide Zoom URL show up as in person meetings

### DISCUSSION THREAD

**Discussion by VickiB11** [code4recovery/12-step-meeting-list#445] [Q&A] [answered]
Some of the meetings in the Georgia area have chosen not to include the Zoom URL in their meeting information.  Instead, they provide only the Meeting ID and Password or a contact person to email to obtain the meeting information.  These meetings are now showing up on the TSML as In Person meetings.  How can I remove that designation?

**Comment by tech2serve** [comment, marked as answer]
TSML v3.11 follows the same standards as the Meeting Guide app. I realize Al Anon doesn't use this app, but the specification must be met to support the majority of our users.

In order for a meeting to have the attendance option of *online*, a valid conference URL or conference phone _must_ be provided. This is how users know how attend the meeting.

The recommended option is to provide a valid Zoom link *or* a phone number (hint) to get the meeting to show up as online. Please remember though that invalid information simply confuses users, so my best suggestion is to educate the groups to provide the Zoom URL.

Keep in mind, natively online meetings expect to have approximate addresses. If you provide a complete specific address in addition to the online information, the meeting will display as being available to attend both in person and online (i.e., hybrid).

**Reply by VickiB11** [reply]
Thanks!

**Reply by Grateful-Viv** [reply]
We've had a couple like you described @VickiB11 and for those I've entered https://us02web.zoom.us/MeetingID where MeetingID is what they gave us. This works to satisfy the TSML plugin that it's an online meeting

Also put in the note section the Meeting ID and password, as well as enter that in the URL notes.
